plugins/tileacq.py

```diff
@@ -553,6 +553,9 @@ def sort_das(self, das, ss):
                         else:
                             logging.warning("Unknown acq stream type for %s", s)
                         break
+                if model.MD_ACQ_TYPE in da.metadata:
+                    # if da is found, no need to search other streams
+                    break
 
             if not da.metadata[model.MD_ACQ_TYPE]:
```

Comment:
Could we drop this block, which is never used?

Reply:
Are you sure it's never used? I thought that I should break from the for outer for loop as well when I found the stream. So with this statement I want to achieve that if I found the statement from the inner loop and break (then MD_ACQ_TYPE is in metadata), then I also break from the outer loop, otherwise (MD_ACQ_TYPE is not added in the inner loop) we just move on to the next stream.

Reply:
Actually, I meant the *next* block (but github automatically changed the position of my comment). This block is indeed useful!

Reply:
I sort of liked it as a fail-safe if the data array for some reason couldn't be found in the streams. This should indeed not happen, but in case something goes wrong, I thought it might help with the debugging process a lot. Still, I see your point, and if you don't like it I'll delete it.

Reply:
I don't mind the warning, but for now, it can logically either do nothing or raise a KeyError. I'd suggest that you move the warning to the else of the for loop.

Reply:
You're right, that's not very useful. I'll change it.